pkg/dependency/parser/nodejs/pnpm/parse.go

```diff
@@ -2,10 +2,12 @@ package pnpm
 
 import (
 	"fmt"
+	"sort"
 	"strconv"
 	"strings"
 
 	"github.com/samber/lo"
+	"golang.org/x/exp/maps"
```

Comment:
```suggestion
	"maps"
```

Reply:
This is weird, but `maps` package doesn't have `Values` and `Keys` functions:
https://pkg.go.dev/maps